static/js/data/faqs.js

```diff
@@ -1,45 +1,256 @@
+export const sectionFAQs = {
+  "General Questions": {
+    "What is OVS?": (
+      <div>
+        OVS is a web site for video hosting and distribution. It was built to
+        host MIT videos that cannot be hosted on other platforms (like YouTube
+        or Vimeo).
+      </div>
+    ),
+    "What kinds of videos does OVS host?": (
+      <ul>
+        <li>Video lecture capture, using a custom multi-angle player</li>
+        <li>
+          Videos used for residential teaching at MIT. Usually the permission to
+          view these videos is limited to the students in a specific class.
+        </li>
+        <li>
+          Videos uses for worldwide teaching that need to be viewed in regions
+          where YouTube is blocked
+        </li>
+        <li>Videos that used to be hosted on techtv.mit.edu</li>
+      </ul>
+    )
+  },
+  "Automated Video Lecture Capture": {
+    "What is Automated Video Lecture Capture?": (
+      <div>
+        <div>
+          Some rooms on campus at MIT have been configured to automatically
+          record lectures using multiple cameras. Currently these rooms include
+          2-131, 2-190, 6-120 and 34-101. Recording requests should be sent to{" "}
+          <a
+            href="https://mvp.mit.edu/schedule/"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            https://mvp.mit.edu/schedule/
+          </a>
+        </div>
+        <div>
+          When viewing the videos recording in these rooms, viewers can select
+          which camera or presentation source they prefer. This avoids the need
+          for a an active camera operator who pans or switches between the
+          instructor, chalkboard, computer display, etc.
+        </div>
+      </div>
+    ),
+    "How can I have my lectures recorded?": (
+      <div>
+        Automated lecture capture is currently available in rooms 2-131, 2-190,
+        6-120 and 34-101. You can request that your lectures be recorded by
+        filling out the form at{" "}
+        <a
+          href="https://mvp.mit.edu/schedule/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          https://mvp.mit.edu/schedule/
+        </a>
+      </div>
+    ),
+    "How long does it take for the lecture capture videos to become available?": (
+      <div>
+        Videos recorded by the lecture capture systems in the classroom are
+        automatically uploaded to the course collection overnight. Collection
+        owners will receive an email notification with a link to each lecture
+        when it is done being processed.
+      </div>
+    )
+  },
+  "Uploading Video and Managing Collections": {
+    "Who can host videos on OVS?": (
+      <ul>
+        <li>Anyone in the MIT community can host videos on OVS.</li>
+        <li>
+          For automated lecture capture, fill out this request form from MIT
+          Video Productions:
```

Comment:
This could use a space after the semi-colon for spacing.

Reply:
will do